scripts/seed_users.js

```diff
@@ -31,10 +31,7 @@ var common = require('../dashboard/helper/common');
 // Import xocolors helper
 var xocolors = require('../dashboard/helper/xocolors')();
 
-// Set the node enviornment to test
-process.env.NODE_ENV = 'test';
-
-// Initiate
+// Initialize settings for sugarizer.ini
 var settings;
 
 // Read settings from sugarizer.ini file
```

Comment:
The user should be able to change ini file. In the targeted deployment there is a different ini file for each school. I suggest to read sugarizer.ini except if the ENV is different, like this: https://github.com/llaske/sugarizer-server/blob/dev/config/settings.js#L9

Reply:
Done:
https://github.com/llaske/sugarizer-server/blob/322c84b89518722eee33513759a5b13bdea94839/scripts/seed_users.js#L40-L42